.travis.yml

```diff
@@ -1,46 +1,31 @@
-language: python
+language: generic
 
 sudo: false
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
 
-python:
-  # We don't actually use the Travis Python, but this keeps it organized.
-  - "2.7"
-  - "3.3"
-  - "3.4"
+matrix:
+  include:
+    - { os: linux, env: PYTHON_VERSION=2.7 }
+    - { os: linux, env: PYTHON_VERSION=3.5 }
+    - { os: linux, env: PYTHON_VERSION=3.6 }
+    - { os: osx, env: PYTHON_VERSION=2.7 }
+    - { os: osx, env: PYTHON_VERSION=3.5 }
+
+#env:
+#  global:
+#  - ENCRYPTION_LABEL: "898e0934cf65"
+#  - COMMIT_AUTHOR_EMAIL: "janos.sallai@vanderbilt.edu"
+
 install:
-  # You may want to periodically update this, although the conda update
-  # conda line below will keep everything up-to-date.  We do this
-  # conditionally because it saves us some downloading if the version is
-  # the same.
-  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      wget http://repo.continuum.io/miniconda/Miniconda-3.4.2-Linux-x86_64.sh -O miniconda.sh;
-    else
-      wget http://repo.continuum.io/miniconda/Miniconda3-3.4.2-Linux-x86_64.sh -O miniconda.sh;
-    fi
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - hash -r
```

Comment:
This is effectively the content of `install_conda.sh`. I personally find it a bit simpler to have it in here - makes it a tad easier for people less familiar with the whole CI toolchain to find where things are happening but your call.